get-members-of-parliament/__init__.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+import azure.functions as func
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    import requests
+    import time
+    import json
+    from bs4 import BeautifulSoup
+    from bs4.dammit import EncodingDetector
+
+    logging.info('Python HTTP trigger function processed a request.')
+    
+    result_list = []
+
+    rooms = [1,2]
+    for r in rooms:
+        link = "http://www.cdep.ro/pls/parlam/structura2015.de?leg=2020&cam={}".format(r)
```

Comment:
Please make 'leg' and 'cam' parameters. 
Also add their values to the root element of the JSON response.
- cam = 1 - get senators
- cam = 2 - get deputies
- cam = 3 - get all members

Reply:
done